### ISSUE TITLE
Question about loss and hyperparameters

### ISSUE THREAD

**Issue by TopTea1** [facebookresearch/swav#69]
Hi, first, thanks for sharing your great work !

I'm trying to train my network using swav in 2x160 + 4x96 setting, I'm using the hyperparameters provided in the bs_256 script. The loss start to decrease, and seems to be stuck after 3-4epochs did I need to adapt the hyperparameters ? Or made some other changes ?

Thanks for your helps

**Comment by Erfun76** [comment]
> Hi, first, thanks for sharing your great work !
> 
> I'm trying to train my network using swav in 2x160 + 4x96 setting, I'm using the hyperparameters provided in the bs_256 script. The loss start to decrease, and seems to be stuck after 3-4epochs did I need to adapt the hyperparameters ? Or made some other changes ?
> 
> Thanks for your helps

Reduce queue length. I encountered this issue in some of my training and using a smaller queue solved the problem.

**Comment by xcvil** [comment]
I had the same issue. During my warm up training, the loss was not reduced anymore and kept the same value.

**Comment by ibro45** [comment]
If I understood correctly, you still keep the same number of prototypes (3000) and batch size (64), while reducing the queue size from 3840 to, e.g., 50? How does that influence the equipartition constraint?
